gulp/tests.js

```diff
@@ -4,26 +4,10 @@ import path from 'path'
 import gutil from 'gulp-util'
 import {paths} from './paths'
 import webdriver from 'gulp-webdriver'
-import selenium from 'selenium-standalone'
 import yargs from 'yargs'
 
 const KarmaServer = karma.Server
 
-export let seleniumServer
-
-export function startSeleniumServer(done) {
```

Comment:
Is this going to cause issues running the functional tests locally? I am getting ERROR: Couldn't connect to selenium server when running the tests locally.